great_expectations/datasource/fluent/databricks_sql_datasource.py

```diff
@@ -80,14 +108,29 @@ class DatabricksSQLDatasource(SQLDatasource):
     Args:
         name: The name of this DatabricksSQL datasource.
         connection_string: The SQLAlchemy connection string used to connect to the postgres database.
-            For example: "databricks+connector://token:<token>@<host>:<port>/<database>?http_path=<http_path>"
+            For example: "databricks://token:<token>@<host>:<port>/<database>?http_path=<http_path>&catalog=<catalog>&schema=<schema>""
         assets: An optional dictionary whose keys are TableAsset or QueryAsset names and whose values
             are TableAsset or QueryAsset objects.
     """
 
     type: Literal["databricks_sql"] = "databricks_sql"  # type: ignore[assignment]
     connection_string: Union[ConfigStr, DatabricksDsn]
 
+    def test_connection(self, test_assets: bool = True) -> None:
+        try:
+            super().test_connection(test_assets)
+        except TestConnectionError as e:
+            nested_exception = None
+            if e.__cause__ and e.__cause__.__cause__:
+                nested_exception = e.__cause__.__cause__
+
+            # Raise specific error informing how to install dependencies only if relevant
+            if isinstance(nested_exception, sa.exc.NoSuchModuleError):
+                raise TestConnectionError(
+                    "Could not connect to Databricks - please ensure you've installed necessary dependencies with `pip install great_expectations[databricks]`."
+                )
```

Comment:
Should this use `from e`? I never know when to use it. Also, I decided to not include the rest of the error trace since it felt verbose but lmk if you disagree

Reply:
Yes exactly.
```suggestion
                raise TestConnectionError(
                    "Could not connect to Databricks - please ensure you've installed necessary dependencies with `pip install great_expectations[databricks]`."
                ) from e
```